src/devices/weten.ts

```diff
@@ -20,22 +20,30 @@ const definitions: Definition[] = [
         fingerprint: tuya.fingerprint('TS0601', ['_TZE204_6fk3gewc']),
         model: 'PCI E',
         vendor: 'WETEN',
-        description: 'PC switch',
+        description: 'Remote Control PCI E Card for PC',
+        whiteLabel: [{vendor: 'Weten', model: 'Tuya PRO'}],
```

Comment:
Could you provide a link to the `Tuya PRO` device?

Reply:
https://sl.aliexpress.ru/p?key=1IDJsOo